src/main/java/com/google/devtools/build/lib/rules/android/AarImport.java

```diff
@@ -395,7 +434,7 @@ private static SpawnAction createAarEmbeddedJarsExtractorActions(
                 .addExecPath("--input_aar", aar)
                 .addExecPath("--output_dir", jarsTreeArtifact)
                 .addExecPath("--output_singlejar_param_file", singleJarParamFile)
-                .build())
+                    .build(), isPersistentAarExtractor ? WORKERS_FORCED_PARAM_FILE_INFO : null)
```

Comment:
same comment about ternary statement, also please fix the extra space introduced on this line.